doc/source/data/package-ref.rst

```diff
@@ -135,6 +135,8 @@ Built-in Datasources
 .. autoclass:: ray.data.datasource.FileBasedDatasource
     :members:
 
+.. autoclass:: ray.data.datasource.ImageFolderDatasource
```

Comment:
```suggestion
.. autoclass:: ray.data.datasource.ImageFolderDatasource
    :members:
```

Reply:
`:members:` would only add `ImageFolderDatasource.prepare_read`. IMO it's better to leave it out

> prepare_read(parallelism: int, paths: Union[str, List[str]], filesystem: Optional[pyarrow.fs.FileSystem] = None, columns: Optional[List[str]] = None, schema: Optional[Union[type, pyarrow.lib.Schema]] = None, meta_provider: ray.data.datasource.file_meta_provider.ParquetMetadataProvider = <ray.data.datasource.file_meta_provider.DefaultParquetMetadataProvider object>, _block_udf: Optional[Callable[[Union[List[ray.data.block.T], pyarrow.Table, pandas.DataFrame, bytes]], Union[List[ray.data.block.T], pyarrow.Table, pandas.DataFrame, bytes]]] = None, **reader_args)→ List[[ray.data.datasource.datasource.ReadTask](https://docs.ray.io/en/master/data/package-ref.html#ray.data.ReadTask)][[source]](https://docs.ray.io/en/master/_modules/ray/data/datasource/parquet_datasource.html#ParquetDatasource.prepare_read)
Creates and returns read tasks for a Parquet file-based datasource.